server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTestHelper.java

```diff
@@ -239,8 +260,49 @@ List<String> modifyGroups(String name, ModifyGroupsRequestV2 request) throws IOE
     });
   }
 
-  Response deleteSeries(String name) throws IOException {
+  Response deleteSeries(String name, String secretDeletionMode) {
+    String uri = UriBuilder.fromPath("/automation/v2/secrets")
+        .path(name)
+        .queryParam("deletionMode", secretDeletionMode)
+        .toString();
```

Comment:
We hand-assemble URIs using String.format() without proper escaping, which is sloppy. This is intended to provide a better example going forward.